### DISCUSSION TITLE
Why Header and Footer of BioDrop is not available in Light Mode ?

### DISCUSSION THREAD

**Discussion by AbhinavTheDev** [EddieHubCommunity/LinkFree#9067] [Ideas]
Hello Everyone,
Today, When I checked my BioDrop Profile so I found something odd about UI of platform so I think i need to share this with all of you. When I Use Dark Mode in BioDrop so it's color scheme is *continous* but when i use light mode so it only applies for main section of the page. 

Light mode should be applicable in header as well. Footer can be dark as it is the end section of the page but **Header must have light mode related with color scheme of platform.**

This will improve the UI of platform because color scheme is really matters. Here I am attaching the pic of whole pages for your references. You can see the difference of colors continuity in screenshots.

![Screenshot 2023-09-14 at 13-48-03 Abhinav](https://github.com/EddieHubCommunity/BioDrop/assets/85792055/91279bb9-cd98-496e-9bea-1fb89ce4a44d)

![Screenshot 2023-09-14 at 13-48-22 Abhinav](https://github.com/EddieHubCommunity/BioDrop/assets/85792055/48126cbc-2c67-48c6-b6ee-ecf27233bf1e)

**Comment by rahulpandey70** [comment]
> Hello Everyone, Today, When I checked my BioDrop Profile so I found something odd about UI of platform so I think i need to share this with all of you. When I Use Dark Mode in BioDrop so it's color scheme is _continous_ but when i use light mode so it only applies for main section of the page.
> 
> Light mode should be applicable in header as well. Footer can be dark as it is the end section of the page but **Header must have light mode related with color scheme of platform.**
> 
> This will improve the UI of platform because color scheme is really matters. Here I am attaching the pic of whole pages for your references. You can see the difference of colors continuity in screenshots.
> 
> ![Screenshot 2023-09-14 at 13-48-03 Abhinav](https://user-images.githubusercontent.com/85792055/267908434-91279bb9-cd98-496e-9bea-1fb89ce4a44d.png)
> 
> ![Screenshot 2023-09-14 at 13-48-22 Abhinav](https://user-images.githubusercontent.com/85792055/267908800-48126cbc-2c67-48c6-b6ee-ecf27233bf1e.png)

I think, this is how `BioDrop` is designed.

**Comment by eddiejaoude** [comment]
Good spot 👍  we are going through an app redesign, so this will all change soon - so I will close for now
